src/hb-ot-color-colr-table.hh

```diff
@@ -198,6 +675,17 @@ struct COLR
     return record;
   }
 
+  const BaseGlyphV1Record* get_base_glyphV1_record (hb_codepoint_t gid) const
+  {
+    if ((unsigned int) gid == 0) // Ignore notdef.
+      return nullptr;
+
+    const BaseGlyphV1Record* record = &(this+baseGlyphsV1List).bsearch ((unsigned) gid);
+    if ((record && (hb_codepoint_t) record->glyphId != gid))
```

Comment:
Many things wrong about this function. The gid==0 check, the extra check to return nullptr. We don't do those things. Remove this function and add back when we need to use it.